var/spack/repos/builtin/packages/apptainer/spack_perms_fix.sh.j2

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -eu
+
+{% for cf  in chown_files %}
+chown root {{ prefix }}/{{ cf }}
+{% endfor %}
+
+{% for sf in setuid_files %}
+chmod 4555 {{ prefix }}/{{ sf }}
+{% endfor %}
```

Comment:
Just to confirm - this is part of the instructions from spack build log that they ask to run?

Reply:
Yes, this file will be populated with stuff from the recipe and saved in `$(spack location -i apptainer)/bin`.

It's maybe easiest to just run this on updates to `apptainer` (after review)